src/translations/ja.translation

```diff
@@ -7644,11 +7648,11 @@
         <name>Batch_Gift_Entry</name>
     </customTabs>
     <customTabs>
-        <label><!-- Contact Merge (OLD) --></label>
+        <label>取引先責任者のマージ</label>
         <name>Contact_Merge</name>
     </customTabs>
     <customTabs>
-        <label>取引先責任者のマージ</label>
+        <label><!-- Contact Merge (OLD) --></label>
         <name>Contact_Merge</name>
```

Comment:
@dmkud This is duplicated, I think we want to keep the Contact_Merge from Line 7652

Reply:
sounds good @bdvorachek